apps/src/lib/tools/jsinterpreter/codegen.js

```diff
@@ -1,4 +1,3 @@
-/* global CanvasPixelArray, Uint8ClampedArray */
 import {dropletGlobalConfigBlocks} from '../../../dropletUtils';
```

Comment:
it's totally been a dumping ground. I need to take a look and see how to reorganize it.